Week7/app/src/main/java/com/kulloveth/covid19virustracker/model/NewsResponse.kt

```diff
@@ -0,0 +1,24 @@
+package com.kulloveth.covid19virustracker.model
+
+import android.os.Parcelable
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import kotlinx.android.parcel.Parcelize
+
+@Parcelize
```

Comment:
As was mentioned today, Filip told us we can use model classes for the different parts of the app, but this model class is working just fine and multi-tasks for all the different uses.